Calibration/HcalCalibAlgos/test/isoTrackRecoAnalysis_cfg.py

```diff
@@ -0,0 +1,86 @@
+import FWCore.ParameterSet.Config as cms
+process = cms.Process('ANALYSIS')
+
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+process.MessageLogger = cms.Service("MessageLogger",
+    destinations = cms.untracked.vstring('warnings','errors',
+                                         'cout','cerr'),
+    categories = cms.untracked.vstring('HcalIsoTrack'), 
+    debugModules = cms.untracked.vstring('*'),
+    warnings = cms.untracked.PSet(
+        placeholder = cms.untracked.bool(True)
+    ),
+    default = cms.untracked.PSet(
+
+    ),
+    errors = cms.untracked.PSet(
+        placeholder = cms.untracked.bool(True)
+    ),
+    cerr = cms.untracked.PSet(
+        optionalPSet = cms.untracked.bool(True),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        noTimeStamps = cms.untracked.bool(False),
+        FwkReport = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            reportEvery = cms.untracked.int32(500),
+            limit = cms.untracked.int32(10000000)
+        ),
+        default = cms.untracked.PSet(
+            limit = cms.untracked.int32(10000000)
+        ),
+        Root_NoDictionary = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            limit = cms.untracked.int32(0)
+        ),
+        FwkJob = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            limit = cms.untracked.int32(0)
+        ),
+        FwkSummary = cms.untracked.PSet(
+            optionalPSet = cms.untracked.bool(True),
+            reportEvery = cms.untracked.int32(1),
+            limit = cms.untracked.int32(10000000)
+        ),
+        threshold = cms.untracked.string('INFO')
+     ),
+     cout = cms.untracked.PSet(
+        threshold = cms.untracked.string('INFO'),
+        noTimeStamps = cms.untracked.bool(True),
+        INFO = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        DEBUG = cms.untracked.PSet(
+            limit = cms.untracked.int32(0)
+        ),
+        HcalIsoTrack = cms.untracked.PSet(
+            limit = cms.untracked.int32(-1)
+       )
+    )
+)
+
+process.load("Configuration.Geometry.GeometryIdeal_cff")
+process.load('Configuration/StandardSequences/GeometryExtended_cff')
+process.load('Configuration/StandardSequences/MagneticField_38T_cff')
+process.GlobalTag.globaltag = 'START53_V15::All'
```

Comment:
Hi

There are 2 workflows: IsoTrack and HBHEMuon. I shall put together the information and the best will be to give a summary by next Monday's meeting. Meanwhile I can send some number to you.

Sunanda
________________________________
From: Marco Musich [notifications@github.com]
Sent: 07 May 2015 18:19
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-alca11 Tune+test several AlCaReco jobs and prepare DQM plots for IsoTrack Trigger (#8943)


In Calibration/HcalCalibAlgos/test/isoTrackRecoAnalysis_cfg.py<https://github.com/cms-sw/cmssw/pull/8943#discussion_r29868383>:

> +        INFO = cms.untracked.PSet(
> +            limit = cms.untracked.int32(0)
> +        ),
> +        DEBUG = cms.untracked.PSet(
> +            limit = cms.untracked.int32(0)
> +        ),
> +        HcalIsoTrack = cms.untracked.PSet(
> +            limit = cms.untracked.int32(-1)
> +       )
> +    )
> +)
> +
> +process.load("Configuration.Geometry.GeometryIdeal_cff")
> +process.load('Configuration/StandardSequences/GeometryExtended_cff')
> +process.load('Configuration/StandardSequences/MagneticField_38T_cff')
> +process.GlobalTag.globaltag = 'START53_V15::All'


@bsunanda<https://github.com/bsunanda> regarding the change in data output rate could we arrange a discussion with some quantitative numbers?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/8943/files#r29868383>.